data/human/human missions.txt

```diff
@@ -6849,3 +6849,308 @@ mission "FW Refugees to Humanika"
+				die
+
+			label "see us"
+			`	"I have no idea," yells Bunker. "Right now we have bigger things to worry about!"`
+			choice
+				`	"Call back the drones immediately."`
+					goto "call drones"
+				`	"I'm sorry but we have to leave the drones behind."`
+
+			label "no can do"
+			`	Bunker hits the recall button for the drones anyway. "This display is a low resolution proxy," explains Bunker. "If we leave the drones we lose all the footage we need. It looks like the Quicksilvers aren't coming straight for us. We have time. The drones are less than two minutes out."`
+			choice
+				`	"We're leaving. Now!"`
+				`	"Fine. Get to the cargo bay and lock down the drones as soon as they land. We take off on your mark."`
+					goto escape
+
+			`	"I can't let you do that, <first>," says Bunker, standing between you and the controls. "The drones are almost here. Just let me go to the cargo bay to lock them down and we're out of here.`
+			choice
+				`	(Charge him with mutiny.)`
+					goto blaster
+				`	(Let him go to the cargo bay and lock down the drones.)`
+					goto escape
+
+			label "call drones"
+			`	"Aye aye, Captain," he says as he hits a button. "Drone ETA: two minutes, fifteen seconds."`
+			`	"Get to the cargo bay," you say. "Lock down the drones as soon as they arrive and get belted in. We'll take off on your mark."`
+
+			label escape
+			`	Less than a minute later, you hear Bunker on the comms. "Drones are locked down and ready to go."`
+			`	The moment you pull out of the canyon, pirate Quicksilvers appear in the distance from all directions. You shunt the <ship>'s repulsors into full blast and accelerate into space with the pirates in hot pursuit.`
+				launch
+
+			label blaster
+			`	You get up and point your pistol at Bunker while ordering him to stand down. "Don't do this," he says. You turn around to look for the handcuffs when you notice a slight hum from outside before getting raked with particle fire from the pirate Quicksilvers, who have found your location and are now speeding towards you. You try to pull out to space but another flotilla of Quicksilvers appear, sending even more volleys of particle fire at you. The <ship>'s repulsors are hit and you feel your pilot seat go out underneath you as your ship falls from the sky, exploding on impact with <origin>'s surface.`
+				die
+
+	npc
+		personality heroic nemesis vindictive launching
+		government "Pirate"
+		ship "Quicksilver" "Don't Run"
+		ship "Quicksilver" "Obey"
+		ship "Quicksilver" "Work Is Freedom"
+		ship "Quicksilver" "Do As You're Told"
+		ship "Quicksilver" "Got You"
+		ship "Quicksilver" "Futile"
+
+	on visit
+		dialog `Your escort carrying the spy drones was left behind. Better take off and wait for them to enter the system.`
+
+
+mission "Adelita 5"
+	name "Slave trade"
+	description "Bring everyone back to <destination>. (You will be paid <payment> after you complete the entire job.)"
```

Comment:
Honestly, I'm not entirely sure if this will display the payment in the `on complete` branch or the apparent payment.

Reply:
I thought the apparent payment would come first, but I'll test it later.

Reply:
It depends on what part of the mission you are in... it's a bit messy when you have both. The wiki says

> `<payment>` = "1 credit" or "N credits," where N is the payment amount from the on complete block (or, beginning in v. 0.10.0, the "apparent payment" of the mission, if one is given), unless the replacement is in an on * block or below a conversation apply node that has its own payment, in which case that on * block's or the most recent apply's payment is used

If I read that correctly, that would indeed show the apparent payment in the description. But yeah, this is complex enough to need testing.

~~I added the damn thing, and I'm still not sure how the fallback thing works exactly.~~

Reply:
<img width="1440" alt="Screenshot 2023-11-09 at 11 32 01 PM" src="https://github.com/endless-sky/endless-sky/assets/115441627/acee2b8f-4cfc-4a47-b764-a1c07618c4cb">
It works.